### ISSUE TITLE
Incorrect type definition for style in Options to interaction/Select

### ISSUE THREAD

**Issue by ejn** [openlayers/openlayers#12594]
Tested with ol@6.6.1

In the generated Typescript definitions for interaction/Select the constructor option key `style` is missing the `null` option.

Documentation (`interaction/Select.d.ts` line 30-):
```
 * @property {import("../style/Style.js").StyleLike|null} [style]
 * Style for the selected features. By default the default edit style is used
 * (see {@link module:ol/style}). Set to `null` if this interaction should not apply
 * any style changes for selected features.
 * If set to a falsey value, the selected feature's style will not change.
```

Code (same file, line 134-):
```
    /**
     * Style for the selected features. By default the default edit style is used
     * (see {@link module:ol/style}). Set to `null` if this interaction should not apply
     * any style changes for selected features.
     * If set to a falsey value, the selected feature's style will not change.
     */
    style?: import("../style/Style.js").default | import("../style/Style.js").default[] | ((arg0: import("../render/Feature.js").default | import("../Feature.js").default<any>, arg1: number) => void | import("../style/Style.js").default | import("../style/Style.js").default[]);
```

This coveres `StyleLike` and `undefined`, but not `null`, which leads to compile errors when attempting to use this.

I'm guessing this is a systematic error that could affect other definitions - although it only showed up for us here: I couldn't find any issues or changes since 6.6.0 that look relevant to this, but maybe I've not yet looked in the right places?

**Comment by ahocevar** [comment]
I found a fix, see #12595.

**Comment by ejn** [comment]
You are correct about using `@types/ol`, but obviously we would prefer to use the inbuilt type definitions for v6.6.0+

Can confirm that the fix looks good, based on looking at the results of a local build of your branch. Thanks @ahocevar !
